### PR TITLE
Chore(web-twig): Introduce docker support for seamless development

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

**Comment by literat** [comment]
@adamkudrna and  @crishpeen Please try it to use and run tests and other commands.

File: packages/web-twig/Makefile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+install: composer
+
+phpstan: ## Run composer phpstan
+	@$(COMPOSER) phpstan
+
+phpunit: ## Run composer phpunit
+	@$(COMPOSER) phpunit
+
+phpunit-coverage: ## Run composer phpunit:coverage
+	@$(COMPOSER) phpunit:coverge
```
Comment (crishpeen) [inline packages/web-twig/Makefile:59]:
```suggestion
	@$(COMPOSER) phpunit:coverage
```

File: packages/web-twig/Makefile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
```
Comment (crishpeen) [inline packages/web-twig/Makefile:60]: Can we add `phpunit-update` as well?

File: packages/web-twig/Makefile [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ecs: ## Run composer ecs
+	@$(COMPOSER) ecs
+
+ecs-fix: ## Run composer ecs-fix
+	@$(COMPOSER) ecs-fix
```
Comment (crishpeen) [inline packages/web-twig/Makefile:65]:
```suggestion
	@$(COMPOSER) ecs:fix
```

**Review by crishpeen — CHANGES_REQUESTED** [review]

File: packages/web-twig/Makefile (hunk 2 above)
Comment (literat) [reply packages/web-twig/Makefile:63]: Added.

File: packages/web-twig/Makefile (hunk 3 above)
Comment (literat) [reply packages/web-twig/Makefile]: Fixed.

File: packages/web-twig/Makefile (hunk 1 above)
Comment (literat) [reply packages/web-twig/Makefile]: Fixed.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**